TALK.md

```diff
@@ -401,6 +399,48 @@ _container.requirements.general_ [Allocator-aware containers] N4687
 
 Containers use the `allocator_traits` template to get information about the allocator.
 
+---
+
+## OPTIONAL(REMOVE): Nullable Pointer - Exposition only
+
+```cpp
+template <typename T>
+concept NullablePointer = requires(T t) {
```

Comment:
This is a named requirement in the standard, so it is not formalised like this but I have tested and used this concept in my own code (against boost offset pointer) and it follows the rules here: https://en.cppreference.com/w/cpp/named_req/NullablePointer.